libs/domains/clusters/feature/src/lib/cluster-installation-guide-modal/cluster-installation-guide-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import download from 'downloadjs'
+import { Button, Callout, ExternalLink, Icon, IconAwesomeEnum } from '@qovery/shared/ui'
+import { useInstallationHelmValues } from '../hooks/use-installation-helm-values/use-installation-helm-values'
+
+export interface ClusterInstallationGuideModalProps {
+  clusterId: string
+  organizationId: string
+  onClose: () => void
+}
+
+export function ClusterInstallationGuideModal({
+  clusterId,
+  organizationId,
+  onClose,
+}: ClusterInstallationGuideModalProps) {
+  const { mutateAsync: getInstallationHelmValues } = useInstallationHelmValues()
+  const downloadInstallationValues = async () => {
+    const installationHelmValues = await getInstallationHelmValues({ organizationId, clusterId })
+    download(installationHelmValues ?? '', `cluster-installation-guide-${clusterId}.yaml`, 'text/plain')
+  }
+
+  return (
+    <div className="p-6 flex flex-col gap-6">
+      <h2 className="h4 text-neutral-400">Installation guide</h2>
+
+      <ol className="ml-4 list-outside list-decimal" type="1">
+        <li className="mb-6">
+          <span className="text-sm font-medium">
+            Save the following yaml, it contains the Qovery configuration assigned to your cluster.
+          </span>
+          <br />
+          <span className="inline-block mt-2">
+            <Button size="lg" variant="outline" color="neutral" onClick={downloadInstallationValues}>
```

Comment:
Missing the props `loading` ?

Reply:
https://github.com/Qovery/console/pull/1117/commits/445d4af32039b46aab294c7c9db79ae0c32b3040